osquery/tables/system/darwin/crashes.cpp

```diff
@@ -88,18 +83,26 @@ void readCrashDump(const std::string& app_log, Row& r) {
       continue;
     }
 
-    if (kCrashDumpKeys.count(toks[0]) == 0) {
+    if (kCrashDumpKeys.count(toks[0]) == 0 &&
+        kRegisters.find(toks[0]) == kRegisters.end()) {
```

Comment:
IIRC, we generally prefer `kRegisters.count(toks[0]) > 0` for existence checking in maps and sets.